src/components/OptionRow.js

```diff
@@ -75,167 +76,191 @@ const defaultProps = {
-                                            size={CONST.AVATAR_SIZE.DEFAULT}
-                                        />
-                                    ) : (
-                                        <MultipleAvatars
-                                            icons={props.option.icons}
-                                            size={CONST.AVATAR_SIZE.DEFAULT}
-                                            secondAvatarStyle={[
-                                                props.optionIsFocused
-                                                    ? StyleUtils.getBackgroundAndBorderStyle(focusedBackgroundColor)
-                                                    : undefined,
-                                                hovered && !props.optionIsFocused
-                                                    ? StyleUtils.getBackgroundAndBorderStyle(hoveredBackgroundColor)
-                                                    : undefined,
-                                            ]}
-                                            avatarTooltips={props.option.isPolicyExpenseChat ? [props.option.subtitle] : avatarTooltips}
-                                        />
-                                    )
-                                )
+                <Hoverable
+                    containerStyles={[
+                        this.props.isDisabled ? styles.userSelectNone : null,
+                    ]}
+                >
+                    {hovered => (
+                        <TouchableOpacity
+                            ref={el => touchableRef = el}
+                            onPress={(e) => {
+                                this.setState({isDisabled: true});
+                                if (e) {
+                                    e.preventDefault();
                                 }
-                                <View style={contentContainerStyles}>
-                                    <DisplayNames
-                                        accessibilityLabel="Chat user display names"
-                                        fullTitle={props.option.text}
-                                        displayNamesWithTooltips={displayNamesWithTooltips}
-                                        tooltipEnabled={props.showTitleTooltip}
-                                        numberOfLines={1}
-                                        textStyles={displayNameStyle}
-                                        shouldUseFullTitle={props.option.isChatRoom || props.option.isPolicyExpenseChat}
-                                    />
-                                    {props.option.alternateText ? (
-                                        <Text
-                                            style={alternateTextStyle}
+                                const selectedOption = this.props.onSelectRow(this.props.option, touchableRef);
+                                // eslint-disable-next-line rulesdir/prefer-early-return
```

Comment:
```suggestion
                                InteractionManager.runAfterInteractions(() => {
                                    if (!(selectedOption instanceof Promise)) {
                                        return;
                                    }
                                    selectedOption.then(() => {
                                        this.setState({isDisabled: this.props.isDisabled});
                                    });
                                });
```

Reply:
Could you please check, if we can use this to avoid disabling the warning?

Reply:
Yes, I checked and verified we can avoid disabling the warning this way.

Reply:
Done